pkg/datasource/columns.go

```diff
@@ -197,3 +205,67 @@ func (ds *dataSource) Columns() (*columns.Columns[DataTuple], error) {
 	}
 	return cols, nil
 }
+
+var defaultFieldAnnotations = map[string]string{
+	ColumnsWidthAnnotation:     "16",
+	ColumnsEllipsisAnnotation:  string(metadatav1.EllipsisEnd),
+	ColumnsAlignmentAnnotation: string(metadatav1.AlignmentLeft),
+}
+
+var annotationsTemplates = map[string]map[string]string{
```

Comment:
What about [syscall template](https://github.com/inspektor-gadget/inspektor-gadget/blob/2b4e5619a0cdb68e1ef2df0f21ea641169e2df94/pkg/types/types.go#L58)?

Reply:
good catch! There are still some templates for ipaddr, etc. that I won't add for now as I think with the new endpoint structure they aren't needed.